examples/seismic/acoustic/wavesolver.py

```diff
@@ -167,8 +169,25 @@ def gradient(self, rec, u, v=None, grad=None, m=None, **kwargs):
         if m is None:
             m = m or self.model.m
 
-        summary = self.op_grad().apply(rec=rec, grad=grad, v=v, u=u, m=m,
-                                       dt=kwargs.pop('dt', self.dt), **kwargs)
+        if checkpointing:
+            set_log_level('ERROR')
```

Comment:
why? perhaps what we should do is to change the verbosity of the checkpointer

Reply:
Yeah, checkpointing prints a huge load of summary, probably not the best way to silence it.